README.md

```diff
@@ -145,31 +394,43 @@ $ ./oran-o2ims start resource-server --help
 Inside _VS Code_ use the _Run and Debug_ option with the `start
 resource-server` [configuration](.vscode/launch.json).
 
-##### Requests Examples
+#### <a name='InfrastructureInventorySubscriptionServerResourceServer'></a>Infrastructure Inventory Subscription Server (Resource Server)
```

Comment:
This section on the inventory subscription server can now be removed.   It has been incorporated directly into the resource server component.